queries.sql

```diff
@@ -34,4 +34,50 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT MAX(escape_attempts) FROM animals;
 SELECT MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth IS BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+
+-- update animals species_id
+UPDATE animals
+SET species_id = CASE
+    WHEN name LIKE '%mon' THEN 1
+    ELSE 2 
+END;
```

Comment:
- [ ] Nice work updating the animals' table to include the `species_id` value :clap: However since we are updating the database, it would be nice to add this code in the `data.sql` file instead of the `queries.sql`